bundle/manifests/observability-operator.clusterserviceversion.yaml

```diff
@@ -690,6 +698,7 @@ spec:
                 - --images=alertmanager=quay.io/prometheus/alertmanager:v0.26.0
                 - --images=prometheus=quay.io/prometheus/prometheus:v2.49.1
                 - --images=thanos=quay.io/thanos/thanos:v0.33.0
+                - --openshift.enabled=true
```

Comment:
Please don't add this here, this will break our e2e tests. We add this flag in the downstream builds

Reply:
Don't worry, I just did not know how this is done properly. It's a temporary change. It's in a commit with a "remove before merge" message :slightly_smiling_face: